packages/scripts/analysis_error.sh

```
@@ -0,0 +1,83 @@
+  
+  # Check if it's a Flutter package (has pubspec.yaml)
+  if [ ! -f "$package_path/pubspec.yaml" ]; then
+    return 0
+  fi
+  
+  PACKAGES_CHECKED=$((PACKAGES_CHECKED + 1))
+  echo "━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━"
+  echo "📦 Checking: $package_name"
+  echo "━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━"
+  
+  cd "$package_path"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)